pandas/core/ops.py

```diff
@@ -703,6 +702,23 @@ def wrapper(left, right, name=name, na_op=na_op):
     return wrapper
 
 
+def _dispatch_to_index_op(op, left, right, index_class):
```

Comment:
We're going to end up using this again for TimedeltaIndex a few bugfixes from now.

Reply:
you can de-privatize. rename to 'dispatch_to_index_operation'. needs a doc-string.